scribus/scpainter.cpp

```diff
@@ -1955,8 +1962,18 @@ void ScPainter::drawText(QRectF area, QString text, bool filled, int align)
 	double ww = 0;
 	double hh = 0;
 	double r, g, b;
-	QString fontName = m_font.family() + " " + m_font.style();
-	cairo_select_font_face(m_cr, fontName.toLatin1(), CAIRO_FONT_SLANT_NORMAL, CAIRO_FONT_WEIGHT_NORMAL);
+
+	if (m_font.isNone())
```

Comment:
since we add & use the following code several times see https://github.com/HOST-Oman/scribus/commit/252b4486d403abfde698e421053f1383a66a6d62
`const QFont &font = QApplication::font();			p->setFont(PrefsManager::instance()->appPrefs.fontPrefs.AvailFonts.findFont(font.family(), font.styleName()), font.pointSizeF());`

I though we can get rid from them and do it once.